fedmsg_genacls.py

```diff
@@ -98,7 +98,8 @@ def delayed_consume():
 
         self.queued_messages.append(msg)
 
-        moksha.hub.reactor.reactor.callLater(self.delay, delayed_consume)
+        moksha.hub.reactor.reactor.callFromThread(
+            moksha.hub.reactor.reactor.callLater, self.delay, delayed_consume)
```

Comment:
.. and that happens [right here](https://github.com/mokshaproject/moksha/blob/dev/moksha.hub/moksha/hub/api/consumer.py#L95-L96).